src/main/java/frc/robot/Constants.java

```diff
@@ -29,18 +33,31 @@ public class Constants {
     *   public static final int kRightMotor2Port = 3; 
     * }
     */ 
+    public static final class ArmConstants {
+        public static final double kP = 0.1;
+        public static final double kI = 0;
+        public static final double kD = 0;
+        public static final double kF = 0;
```

Comment:
the above 4 variables have very non-descript names

what is P? what is F?

Reply:
p is proportional, i is integral, d is derivative, and f is feedforward. they are the standard constant names for almost all FRC teams for using PID